Hspi/html/feature.js

```diff
@@ -0,0 +1,40 @@
+function getUrlParameterOrEmpty(key) {
+	return getUrlParameter(key) ?? '';
+};
+
+function getUrlParameter(sParam) {
+    var sPageURL = window.location.search.substring(1),
+        sURLVariables = sPageURL.split('&'),
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/dk307/HSPI_InfluxDBPersistence/pullRequest?prid=8479377)
```suggestion
        sURLVariables = sPageURL.split("&"),
```